docs/pages/index.tsx

```diff
@@ -1,47 +1,73 @@
 import type { NextPage } from 'next';
```

Comment:
`import type` is causing formatting to fail. 
```suggestion
import { NextPage } from 'next';
```